homeassistant/components/epion/coordinator.py

```diff
@@ -0,0 +1,45 @@
+"""The Epion data coordinator."""
+
+import logging
+
+from epion import Epion, EpionAuthenticationError, EpionConnectionError
+
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import REFRESH_INTERVAL
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class EpionCoordinator(DataUpdateCoordinator[dict]):
```

Comment:
```
class EpionCoordinator(DataUpdateCoordinator[dict[..]]):
```
A dict of what? Can we type it?

Reply:
Updated to dict[str, Any], as this is what the API / lib currently returns